src/rest/api-request.js

```diff
@@ -0,0 +1,27 @@
+import fetch from 'node-fetch';
+global.Headers = fetch.Headers;
```

Comment:
I am against assigning something to global. I would rather use `new fetch.Headers()`. With TS we would be able to import Headers directly anyway